suite-common/wallet-core/src/fiat-rates/fiatRatesThunks.ts

```diff
@@ -86,7 +87,19 @@ type UpdateCurrentFiatRatesThunkPayload = {
 
 export const updateFiatRatesThunk = createThunk(
     `${fiatRatesActionsPrefix}/updateFiatRates`,
-    async ({ ticker, localCurrency, rateType }: UpdateCurrentFiatRatesThunkPayload) => {
+    async (
+        { ticker, localCurrency, rateType }: UpdateCurrentFiatRatesThunkPayload,
+        { getState },
+    ) => {
+        if (ticker.tokenAddress) {
+            const tokenDefinitions = selectTokenDefinitions(ticker.symbol)(getState());
```

Comment:
It would be better to use `selectSpecificTokenDefinition` so then the return can be more obvious with just `!tokenDefinitions`

Reply:
actually it cant, but i will redo it to use `token-definitions` from config

Reply:
6ae7d78123563d0bc24bee44c4a74553e576fdbe

It would be nice to type whole networks config but let's leave it for another time